README.md

```diff
@@ -74,3 +144,17 @@ ruff format
 ```
 
 **Or** [integrate](https://docs.astral.sh/ruff/integrations/#vs-code-official) it with your favorite code editor (for example, [VSCode](https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff))
+
+## Results
+_Comparison of selector based on our model (AI) with the best selector of V#. All except coverage provided for methods with equal coverage._
```

Comment:
```suggestion
_Comparison of the selector, based on our model (AI), with the best selector of V#. All metrics except coverage provided for methods with equal coverage._
```

Reply:
it is correct either way, so commas indeed can be dropped

Reply:
I've already done it.